app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected_result", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
```

Comment:
It seems that the test cases for (0, 0, [0, 0]) and (14, 14, [0, 0]) are testing the same scenario where both cat and dog ages result in [0, 0]. According to the checklist, make sure that your tests don't test the same case multiple times.